blocks/trending-news/trending-news.js

```diff
@@ -1,10 +1,23 @@
-import { getHostUrl } from '../../scripts/mockapi.js';
 import {
-  Viewport, createPictureElement, observe, fetchData,
+  getResearchAPIUrl, postFormData,
+  Viewport, createPictureElement, observe, parseResponse,
 } from '../../scripts/blocks-utils.js';
 import { decorateIcons, fetchPlaceholders, readBlockConfig } from '../../scripts/aem.js';
 
 const placeholders = await fetchPlaceholders();
+const ICICI_DIRECT_NEWS_HOST = 'https://www.icicidirect.com/research/equity/trending-news/';
+const ICICI_NEWS_THUMBNAIL_ICICI_HOST = 'https://www.icicidirect.com/images/';
+const ICICI_NEWS_THUMBNAIL_FINOUX_HOST = 'http://icicidirect.finoux.com/images/';
```

Comment:
@ravverma  may be we should extract these into block utils

Reply:
@ravverma for http://icicidirect.finoux.com , we can fetch the images from edge worker as done it here
https://github.com/aemsites/icicidirect/pull/148/files#diff-042cbe0e2ef3c10e1d0e56d1416e393d9eca4b7fcdbe69ea03322e7b14303d97R106.

and for https://www.icicidirect.com IMO we should get a confirmation that this is really the correct way. We should be assuming that all this is from UAT environment of their and hence that should be the host url.
We can ask again.

Reply:
Why do we need these? Isn't the api response returning absolute URLs? If not, we should ask.

Reply:
No, API doesn't retun absolute , not even relative it is just return resource part of url only.

Reply:
yes , I will add in tracker